.eslintrc.json

```diff
@@ -15,10 +15,7 @@
     }]
   },
   "parserOptions": {
-    "sourceType": "script",
-    "ecmaFeatures": {
-      "impliedStrict": false
```

Comment:
I can vaguely remember nodejs complaining about missing use stricts. Is this no longer an issue because we require a higher nodejs version now?

Reply:
It's probably due to some removed embedded 3rd-part libs (concat-source-map, ...), not by this PR though.

Some of those libs might not work in strict mode.